pgmpy/utils/utils.py

```diff
@@ -98,25 +103,74 @@ def get_example_model(model):
         "pathfinder": "utils/example_models/pathfinder.bif.gz",
         "pigs": "utils/example_models/pigs.bif.gz",
         "munin": "utils/example_models/munin.bif.gz",
-        "ecoli70": "",
-        "magic-niab": "",
-        "magic-irri": "",
-        "arth150": "",
+        "ecoli70": "utils/example_models/ecoli70.json",
+        "magic-niab": "utils/example_models/magic-niab.json",
+        "magic-irri": "utils/example_models/magic-irri.json",
+        "arth150": "utils/example_models/arth150.json",
         "sangiovese": "",
         "mehra": "",
     }
 
-    if model not in filenames.keys():
-        raise ValueError("dataset should be one of the options")
-    if filenames[model] == "":
-        raise NotImplementedError("The specified dataset isn't available.")
+    if model not in filenames:
+        raise ValueError(f"Unknown model name: {model}")
 
     path = filenames[model]
-    ref = files("pgmpy") / path
-    with gzip.open(ref) as f:
-        content = f.read()
-    reader = BIFReader(string=content.decode("utf-8"), n_jobs=1)
-    return reader.get_model()
+
+    # Determine the model type
+    if model in cat_models:
+        # Handle reading of categorical models
+        if path.endswith(".bif.gz"):
+            from pgmpy.readwrite import BIFReader
+
+            ref = files("pgmpy") / path
+            with gzip.open(ref) as f:
+                content = f.read()
+            reader = BIFReader(string=content.decode("utf-8"))
+            return reader.get_model()
+    elif model in cont_models:
+        from pgmpy.factors.continuous import LinearGaussianCPD
+        from pgmpy.models import LinearGaussianBayesianNetwork
+
+        # Handle reading of continuous models
```

Comment:
We don't need this if statement. Because it's only continuous model, it will always be a JSON.